hw/ip/aes/lint/aes_lint_cfg.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
```

Comment:
these individual .hjson files per lint target seem unfortunate, since they are so
simple and repetitive. I've put some time in the Silicon meeting to discuss if there
is a better way. @msfschaffner @imphil @sriyerg

Reply:
we decided to introduce support for inline lint configurations in the master config file that lists all the cores, hence these files will not be needed anymore.

Reply:
The most recent patch in https://github.com/lowRISC/opentitan/pull/1640 enables to specify the lint targets in-line in the batch list, hence obviating the need for all these separate files. Thanks for adding support for that, @sriyerg!